app/models/skill.rb

```diff
@@ -0,0 +1,8 @@
+class Skill < ApplicationRecord
+  belongs_to :user
+  validates_presence_of :power, :dexterity, :instinct, :stamina, :experience
+  validates :power, :dexterity, :instinct, numericality: { only_integer: true }
+  validates :stamina, :experience, numericality: { only_integer: true }
```

Comment:
4, 5 lines: seems to be the same kind of validation, why did you split them?

Reply:
Maximum length is 80 symbols in line(in accordance with rubocop). If I'll type that in one line rubocop will show it as mistake.